simanneal/anneal.py

```diff
@@ -279,9 +279,8 @@ def run(T, steps):
             step += 1
             dE = self.move()
             if dE is None:
-                T = abs(self.energy() - E)
-            else:
-                T = abs(dE)
+                dE = self.energy() - E
+            T = abs(dE)
```

Comment:
I don't think there's any reason to call `abs()` here, given that the loop condition is `while T == 0.0`.

Reply:
Should be the same behavior as before wrt `T` but I like this approach as `dE` can be used later in the code and have semantically-accurate values regardless of the return value of `move()`. :+1: